### PR TITLE
Added Sky130 example config file and comparison with OpenRAM SRAMs

### PR DESCRIPTION
**Pull request by lakshmi-sathi**
Added an example config file for sky130 (creates 1024x32 & 64x32 fakerams).
In the README, Added a small comparison between the generated fakerams and the standard SRAMs compiled using OpenRAM.
#11 
@taylor-bsg 
@dpetrisko

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,52 @@ All of the generated files can be found in the `./results` directory. Inside
 this directory will be a directory for each SRAM which contains the .lef, .lib
 and v file (as well as some intermediate files used for Cacti).
 
+### Comparison with standard SRAMs generated with OpenRAM compiler
+
+#### Generated Fakerams (Eg:- fakeram130_1024x8)
+
+<img src="https://user-images.githubusercontent.com/58559090/139503688-22503f55-b565-4e00-8f3e-af25aa50d3b1.png" width="800">
```
Comment (mithro) [inline README.md:91]: These images will stop working. You should include the images in this repository and use `fakeram130_1024x8` in your file.

Comment (lakshmi-sathi) [reply README.md:91]: Thanks for pointing that out! 
I will make that change.

**Comment by lakshmi-sathi** [comment]
Included the images in the repository (docs/images) and inserted them in the README using '!()[]'

**Comment by taylor-bsg** [comment]
@stdavids can you review

**Comment by stdavids** [comment]
Sorry for the delay. Thank you @lakshmi-sathi for the contributions!

For posterity I'll state that the PR looks good and I approve.

**Pull request merged**